src/types/result.rs

```diff
@@ -30,6 +30,7 @@ pub enum CheckStatusReasonType {
     Timeout,
     DnsError,
     Failure,
+    SslError,
```

Comment:
If we add to this we'll need to update the schemas

https://github.com/getsentry/sentry-kafka-schemas/blob/main/schemas/uptime-results.v1.schema.json#L6-L11

Reply:
Even just having the better error message now is going to be helpful!